app/serializers/parent_profile_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# JSON API serialization for an OpenSCAP Profile
+class ParentProfileSerializer < ProfileSerializer
```

Comment:
This is required as a quirk of the way FastJsonapi works with `belongs_to :parent_profile`